### DISCUSSION TITLE
Request to Publish Extension on open-vsx.org

### DISCUSSION THREAD

**Discussion by SendToNull** [lucono/karma-test-explorer#98] [General]
Hello,

Thank you for developing the Karma Test Explorer extension. I noticed that the extension is available on the Microsoft Marketplace but not on [open-vsx.org](http://open-vsx.org/), which is an open-source alternative that serves the broader community, including VS Codium users.

Would you consider publishing your extension on [open-vsx.org](http://open-vsx.org/) as well? It would benefit users who prefer or are required to use open-source software. The process is outlined here: [Publishing Extensions](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions).

Thank you for considering this request.

Regards,
Tony

**Comment by Suryachandra-DTH** [comment]
Hello Team,

Any updates?

Thanks & Regards,
Surya

**Comment by SabikAbtahee** [comment]
Did you try downloading the vsix file and then installing it in other ides. I use cursor and I installed many vscode extension this way, which are not available in open-vsx
